Ports/aria2/patches/0002-Include-sys-select.h.patch

```diff
@@ -0,0 +1,24 @@
+From 839f9ff4a68a40d77e873a63531394eb4af8e50b Mon Sep 17 00:00:00 2001
```

Comment:
Please regenerate this using a current version of `./package.sh dev` to get rid of the unnecessary metadata.